linux_os/guide/system/software/integrity/disable_prelink.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora
```

Comment:
This just needs to be updated to have a different directory, but that is out of the scope of this PR.  

http://manpages.ubuntu.com/manpages/trusty/man8/prelink.8.html
https://software.opensuse.org/package/prelink